action.yml

```diff
@@ -27,17 +27,15 @@ inputs:
   registry_password:
     description: "The registry user password"
     required: false
+  force_create:
+    description: |
+      "Pass --force to kn create to replace the existing service,
```

Comment:
sorry, let me try this one again.
\
"Pass `--force` to `kn create`. If the service already exists, the service will be replaced, instead of `kn create` failing.

Also I think you said this before, but this has no effect if `operation` is not `create`, right? let's make that clear here

Reply:
yup, I removed that as we are already mentioning `kn create` here.

Reply:
I think it's good for clarity to say "This input has no effect if `service_operation` is not `create`".

Reply:
Okay :+1: